Neos.Flow/Classes/Http/AbstractMessage.php

```diff
@@ -434,7 +434,7 @@ public function withHeader($name, $value)
     public function withAddedHeader($name, $value)
     {
         $newMessage = clone $this;
-        $newMessage->setHeader($name, $value);
+        $newMessage->setHeader($name, $value, false);
```

Comment:
D'uh, there was also another Bug regarding PSR7 compatibility... `withAddedHeader()` behaved the same as `withHeader()` and overwrote existing ones

Reply:
Is this the right PR to change this - seems more of a standalone PR also in terms of changelog for next releases? And what about tests?

Reply:
Yep. And yep. I'm laaaaaazy 😂